### DISCUSSION TITLE
How to Pause and resume LORA training?

### DISCUSSION THREAD

**Discussion by Konvoname** [bmaltais/kohya_ss#321] [Q&A] [answered]
I try to train about 100+ image but it take 12+ hrs, So I want to pause and resume it later, Is there any solution.

**Comment by quasiblob** [comment]
I'm also interested about this.

**Reply by sebastiandoesart** [reply]
> I'm also interested about this.
When I click the display of the command prompt window, the training process pauses. Then when I want to resume I just go back to the command prompt window, click on the displayed area and press enter, then it resumes.

**Comment by bmaltais** [comment, marked as answer]
Unfortunately it is not supported by kohya's code at the moment

**Comment by andupotorac** [comment]
Would love for this to become possible too.

**Comment by 2blackbar** [comment]
doods, you are aware of the fact that you can resume training from existing lora model, right ? ..... right ? its up there one of the first options in lora panel.... i used it to resume a couple of times
So to resume, save your lora every whatever steps or epoch

**Reply by andupotorac** [reply]
Right, but it's not the same thing. Sometimes you want to pause it, work on something else, then resume it when you have free GPU.

**Reply by somenewaccountthen** [reply]
Does that mean i can use other peoples lora as a base?
